examples/hooks/components/layout/Tabs.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
+import { cx } from '../../cx';
+
+import './Tabs.css';
+
+export type TabProps = {
+  children: React.ReactNode;
+  title: string;
+};
+
+const getTabKey = (index: number, suffix?: string) =>
+  [`tab-${index}`, suffix].filter(Boolean).join('-');
+
+export function Tabs({ children }) {
+  const [currentTab, setCurrentTab] = useState(0);
+  const tabsRefs: HTMLElement[] = [];
```

Comment:
Since this is a plain array, doesn't it get reconstructed at each render?

Maybe you could have something like this instead:

```js
const tabsRef = useRef(React.Children.map((child) => null));
```

Then assign each ref per tab by index?